pandas/core/frame.py

```diff
@@ -832,12 +832,12 @@ def itertuples(self, index=True, name="Pandas"):
                 itertuple = collections.namedtuple(name,
                                                    fields + list(self.columns),
                                                    rename=True)
-                return map(itertuple._make, zip(*arrays))
+                return map(itertuple._make, zip(*iterators))
             except Exception:
                 pass
 
         # fallback to regular tuples
-        return zip(*arrays)
+        return zip(*iterators)
```

Comment:
Sorry, I'm confused. Inside ``frame.py``, you are only changing a variable name, right? How does this relate to fixing #20783?

Reply:
OK, I think I'm no more confused. ``iterators`` does _not_ contain iterators, it just contains ordinary ``Series``. #20783 is fixed, I think, only thanks to the fact that ``iloc`` happens to not make a copy of data. But I would not change this variable name, I would mention what is happening in a comment, and I would try to make sure that this happens reliably.